modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/UriPartsProcessor.java

```diff
@@ -56,8 +67,11 @@ public boolean getKeepOriginal() {
 
     @Override
     public IngestDocument execute(IngestDocument ingestDocument) throws Exception {
-        String value = ingestDocument.getFieldValue(field, String.class);
+        String value = ingestDocument.getFieldValue(field, String.class, ignoreMissing);
 
+        if (Objects.isNull(value)) {
```

Comment:
A nit, but `value == null` is better in this context -- `Objects.isNull` is for passing around method references, not for, like, calling.

Reply:
```suggestion
        if (null == value) {
```

Reply:
Hey @joegallo, it is fixed.
Thank you,